lib/config_cat/cache_policy/helpers.ex

```diff
@@ -3,27 +3,30 @@ defmodule ConfigCat.CachePolicy.Helpers do
 
   alias ConfigCat.CachePolicy
   alias ConfigCat.ConfigCache
-  alias ConfigCat.ConfigFetcher
 
   @type state :: %{
           :cache => module(),
           :cache_key => ConfigCache.key(),
           :fetcher => module(),
-          :fetcher_id => ConfigFetcher.id(),
-          :name => CachePolicy.id(),
+          :instance_id => ConfigCat.instance_id(),
           :offline => false,
           optional(atom()) => any()
         }
 
-  @spec start_link(module(), CachePolicy.options(), map()) :: GenServer.on_start()
-  def start_link(module, options, additional_state \\ %{}) do
```

Comment:
Only one of the cache policies needed the additional state, so I removed it from here and added it to that cache policy's `init` callback instead.